### ISSUE TITLE
Would this be faster on a pre-built container?

### ISSUE THREAD

**Issue by davorg** [davorg/dashboard#42]

**Comment by szabgab** [comment]
Looking at the [detailed report](https://github.com/davorg/dashboard/actions/runs/3803904184/jobs/6470677532) of the most recent run out of the 1m 3 sec total time, 41 sec is "Initialize containers" - that would not go away and only 5 sec is install modules. 

My understanding that you would gain max 5 sec speedup with a pre-built container.

**Comment by davorg** [comment]
Yes, I think you're right. I just raised the ticket to make myself look into it.
